.github/CODEOWNERS

```
@@ -0,0 +1,19 @@
+# Copied files:
+
+.github/dco.yml @IBM/cloudant-sdks
+scripts/setup_couch.sh @IBM/cloudant-sdks
+scripts/setup_wiremock.sh @IBM/cloudant-sdks
+stubs/mappings.json @IBM/cloudant-sdks
```

We might want to select the whole folder:
```suggestion
.github/** @IBM/cloudant-sdks
scripts/** @IBM/cloudant-sdks
stubs/** @IBM/cloudant-sdks
```

---

I think it is not necessary, the scope of this task to add the files that are generated somehow. `.github` has already some language-specific files in it. `scripts`, `stubs` might have as well. But we can turn back to this any time when we feel it is necessary.

---

I think it is necessary. What if a new file is added to these directories? Or a Whole new directory can be added to these folders and we do not get notification about it.

---

I'm happy with keeping this as-is after our Slack call.

---

Yeah, we can keep it as-is.